starter-code/CSS + HTML DONE/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.min.css">
+  <link rel="stylesheet" href="style.css">
+  <script type="text/javascript" src="index.js"></script>
+  <title>Ironhack cart</title>
+</head>
+<body>
+  <div class="container">
+    <h1>Ironhack Merchandising Shop</h1>
+
+    <div class="product">
+      <span class="product-name">IronBubble-Head</span>
+      <span class="cost-of-unit">$25.00</span>
+      <div class="quantity">
+        <label for="qty">QTY</label>
+        <input id="qty-input" type="number" name="quantity" value="0" />
```

Comment:
Little tip, if you don't want to allow negative values, do:
```
<input id="qty-input" type="number" min="0" name="quantity" value="0" />
```
Keep in ming this is not a perfect solution.